configs/config.dev.yaml

```diff
@@ -135,6 +135,25 @@ sender:
   ## slaves to be cycled through
   is_master: True
 
+  # default_rate_def:
+  #   # hard quota 60 messages in a period, soft quota 30 messages in a period, 1 minute HQ period length, 1 minute SQ period length, no plans/targets
+  #   #self.default_rate_def = (40,30,1,1,3600,None,None)
```

Comment:
should this be 60 not 40 to be consistent with comment above?

Reply:
that was a comment I was using for reference and missed taking out, removed